README.md

```diff
@@ -176,6 +174,7 @@ cat tx_swap_signed.json
 # Encode the signed tx
 liquidityd tx encode tx_swap_signed.json
 ```
+
 ### 2.2 Broadcaste transactions using REST APIs
 
 An example of broadcasting transactions using REST API (via gRPC-gateway) can be found in this [link](https://github.com/cosmos/cosmos-sdk/blob/master/docs/migrations/rest.md#migrating-to-new-rest-endpoints). Note that API server should be enabled in `$HOME/.liquidityapp/config/app.toml` to test this.
```

Comment:
```suggestion
For an example of broadcasting transactions using REST API (via gRPC-gateway), see Cosmos SDK [Migrating to New REST Endpoints](https://github.com/cosmos/cosmos-sdk/blob/master/docs/migrations/rest.md#migrating-to-new-rest-endpoints). Testing requires that the API server is enabled in `$HOME/.liquidityapp/config/app.toml`.
```

Reply:
instead of "link" let's use descriptive phrases that provide context for the material that we're linking to, per https://developers.google.com/style/link-text